### PR TITLE
added readme and seed file

### PR DESCRIPTION
**Pull request by MuhammadAbdullah191**
- added readme and seedfile

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+question3 = Question.create(title: "You’re having an animated discussion with a colleague regarding a project that you’re in charge of.\r\nYou:")
+question3.options.create(title: "Don’t dare contradict them", option_type: 0)
+question3.options.create(title: "Think that they are obviously right", option_type: 0)
+question3.options.create(title: "Defend your own point of view, tooth and nail", option_type: 1)
+question3.options.create(title: "Continuously interrupt your colleague", option_type: 1)
+question4 = Question.create(title: "You are taking part in a guided tour of a museum.\r\nYou:" )
+question4.options.create(title: "Are a bit too far towards the back so don’t really participate", option_type: 0)
+question4.options.create(title: "Follow the group without question", option_type: 0)
+question4.options.create(title: "Make sure that everyone is able to hear properly", option_type: 1)
+question4.options.create(title: "Are right up the front, adding your own comments ", option_type: 1)
```
Comment (husnainrazadevsinc) [inline db/seeds.rb]: Please add necessary line-breaks and spaces.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,39 @@
...
+Personality Test app where user can take test to find out their peronslity type
 
-Things you may want to cover:
+Admins can sign in to add question and their options for personality test
 
-* Ruby version
+Devise is usad for admin authentication
 
-* System dependencies
+Added functionality of dynamic screens for personality test where user will be able to navigate through questions one by one
```
Comment (husnainrazadevsinc) [inline README.md]: let's discuss readme file structure in the meeting?

**Review by husnainrazadevsinc — CHANGES_REQUESTED** [review]
Suggested some changes. Please check 👍

File: README.md (hunk 2 above)
Comment (MuhammadAbdullah191) [reply README.md]: sure

**Pull request merged**